src/Compute/Compute/help/Set-AzDiskImageReference.md

```diff
@@ -67,7 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -Id
-Specifies the ID.
+Specifies the ID of PIR or user image.
```

Comment:
Hi @grizzlytheodore , please update the description of ID and Lun in codes as well.

Reply:
they are already updated

Reply:
Please update their help messages as well.

https://github.com/Azure/azure-powershell/blob/7d4f04442de9140bcc5bc845703623fb964ea9cc/src/Compute/Compute/Generated/Disk/Config/SetAzureRmDiskImageReferenceCommand.cs#L43-L49

https://github.com/Azure/azure-powershell/blob/7d4f04442de9140bcc5bc845703623fb964ea9cc/src/Compute/Compute/Generated/Disk/Config/SetAzureRmDiskImageReferenceCommand.cs#L50-L54

Reply:
done, as you requested